### DISCUSSION TITLE
Git LFS included in git install on 2.36.0 but not 2.36.1

### DISCUSSION THREAD

**Discussion by broomhilda99** [git-for-windows/git#3898] [Q&A] [answered]
When I installed git version 2.36.0 it seemed to include git LFS as well, and the release notes also indicated that it did, so I figured it was no longer necessary to install LFS separately.  However, when I installed 2.36.1 on another computer, it did not include git LFS. Was this just a one-time thing for 2.36.0, or will LFS be included with git in the future?

**Comment by dscho** [comment, marked as answer]
Git LFS should still be included by default. Did you (un-)check the Git LFS checkbox when installing? (`/etc/install-options.txt` should also be informative.)

**Reply by broomhilda99** [reply]
Thanks so much for the reply. It was indeed pilot error, sorry for the stupid question. I must have clicked through without noticing the option or that it was unchecked. I was upgrading from a much older version of git -- I wonder if because of that, it was unchecked by default, otherwise it must have been a stray click.

**Reply by dscho** [reply]
No worries! The upgrade from an old version (that might not even have included Git LFS) would explain this, indeed!
